distro.py

```diff
@@ -738,8 +747,9 @@ def __init__(
                 self.os_release_file = usr_lib_os_release_file
 
         self.distro_release_file = distro_release_file or ""  # updated later
-        self.include_lsb = include_lsb
-        self.include_uname = include_uname
+        self.include_lsb = include_lsb if not root_dir else False
+        self.include_uname = include_uname if not root_dir else False
+        self.include_third_bins = include_third_bins if not root_dir else False
```

Comment:
One option for unprivileged execution of binaries from a chroot is [proot](https://proot-me.github.io/). It would only work for binaries from a compatible architecture (e.g. x86 on x86_64 may work or not, depending on kernel config) and any data that app gets from the live kernel would be from the host kernel, not the chroot kernel. And it would need to be installed. Running commands from the chroot without any chroot-like mechanism seems troublesome, yes.